eiffel-syntax-and-usage/glossary.md

```diff
@@ -0,0 +1,65 @@
+<!---
+   Copyright 2020 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+--->
+
+# Eiffel Glossary
+This a non-exhaustive, alphabetical list of terms used in the Eiffel Vocabulary and its documentation.
+
+ - [Terms Used in Event Definitions](#terms-used-in-event-definitions)
```

Comment:
Should ConfidenceLevel also be included in this list?

Reply:
Will be